strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-authorization.yaml

```diff
@@ -2,18 +2,7 @@ authorizationConfiguration:
   roles:
     - name: ANONYMOUS_ROLE
       description: Common anonymous user role
-      privileges:
```

Comment:
@ogryniuk you shouldn't drop this `privileges`, only root `privileges` attribute should be dropped